lib/Crust/Middleware/Static.pm6

```diff
@@ -51,6 +51,6 @@ method !handle-static(%env) {
     }
 
     temp %env<PATH_INFO> = $path;
-    my @res = $!file.(%env);
+    my @res = await $!file.(%env);
```

Comment:
* M::Static needs to check the status code In the CALL-ME method.
* I want to rewrite PATH_INFO in the scope.